openedx/features/edly/utils.py

```diff
@@ -108,3 +108,28 @@ def get_enabled_organizations(request):
         return []
 
     return [studio_site_edx_organization]
+
+
+def create_edly_user_profile(request, user):
```

Comment:
This method is not creating `edly_user_profile`, it is created automatically due to `1-1` relation
rename this to `create_user_link_with_edly_sub_organization`

Reply:
Done